src/WebJobs.Extensions.DurableTask/Correlation/ITelemetryActivator.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Correlation
+{
+    /// <summary>
+    /// ITelemetryActivator is an interface.
+    /// </summary>
+    public interface ITelemetryActivator
```

Comment:
If we want to rename this, we have to do it now, since it is a public interface.

Reply:
I don't think this is a bad name, but I don't think this needs to be `public`, right? Looks to me like it should be safe to make it `private`.

Reply:
Judging by Tsuyoshi's new comments, it sounds like this should remain public if we want to support customers using alternative telemetry protocols. So, if that's a feature we want to support (and I think we should), then this can stay public.

As for the name, I really think this is _fine_. Or were you thinking of some other name?

Reply:
I thought I remembered it being something you hadn't like, but I could be thinking of a different naming scheme. I don't have a big problem with it.